doc/interfaces-vrrp.scd

```diff
@@ -0,0 +1,199 @@
+interfaces-vrrp(5)
+
+# NAME
+
+*interfaces-vrrp* - VRRP extensions for the interfaces(5) file format
+
+# DESCRIPTION
+
+VRRP stands for Virtual Router Redundancy Protocol. This protocol 
+is used to allow multiple backup routers on the same segment to take 
+over operation of each others’ IP addresses if the primary router fails. 
+This is typically used to provide fault-tolerant gateways to hosts 
+on the segment.
+
+Linux has support for MACVLAN from version *3.0*, but the _protodown_ 
+property appears only in kernel *5.1*. Don't try to use this setup for
+kernels older than *5.1*. It will not work.
+
+To be able to use VRRP you need to run an user-space software that 
+manages the Master-Backup setup. A good example is FFRouting software 
+which includes the VRRP daemon.
+
+*Note*: for each type of traffic *IPv4* or *IPv6* you need a complete different 
+interface. For one _main interface_ where you have dual-stack traffic 
+you will need 2 VRRP interfaces: one for IPv4 and one for IPv6.
+For this reason 2 addtitional interfaces are needed because each protocol 
+needs its separate MAC address for that specific traffic. That's why you 
+can't use only one VRRP interface for both IPv4 and IPv6 traffic.
+
+Convention of the name is *vXXXXXXXX-YY-V*, where *XXXXXXXX* is CRC32 value in HEX 
+calculated from main *interface name*. *YY* is the *VRRP ID*. *V* stands for protocol 
+version: *4* for *IPv4* and *6* for *IPv6*.
+
+There are 2 ways to use CRC32: crc32sum tool that is provider with ifupdown-ng 
+or cksum provided by coreutils. The script is searching first for crc32sum and 
+then for cksum.
+
+The tools are having compatible output. crc32sum provided in ifupdown-ng is a stripped 
+version of cksum from coreutils.
+
+This executor takes care about the VRF of the VRRP interfaces in case if 
+main interface should be under a VRF.
+
+See *https://www.kernel.org/doc/html/latest/networking/ipvlan.html* or 
+*https://developers.redhat.com/blog/2018/10/22/introduction-to-linux-interfaces-for-virtual-networking#macvlan* or 
+*http://docs.frrouting.org/en/latest/vrrp.html* for more details.
+
+# VRRP-RELATED OPTIONS under main interface
+
+*vrrp-id* _vrrp id array_
+	List of VRRP _id_. This is used to create the virtual MACVLAN 
+	interface with a specific MAC address for each VRRP instance. 
+	This should match in the FRRouting setup.++
+*Mandadory*: _yes_
+
+*vrrp-address* _array of addresses_
+	The format should be: *VRRP_ID* _IP/MASK_ _IP/MASK_ _.._ *VRRP_ID* _IP/MASK_ _.._++
+Optionally you can assign one or more IPv4 or IPv6 to each VRRP interface.++
+But because each main interface can have multiple VRRPs, we need to specify 
+	on which VRRP pair we want the IP address. The script automatically detects 
+	the type of IP version.++
+*Mandatory*: _no_++
+*Default*: _none_
+
+# EXAMPLES using crc32sum tool and FRRouting to configure the IP interfaces
```

Comment:
There should be one EXAMPLES heading, with the different examples as subheadings.

Reply:
Fixed in the new PR.